.github/workflows/deploy-docs.yml

```
@@ -37,30 +37,26 @@ jobs:
-
-      - name: Setup Python
-        uses: actions/setup-python@v5
+      - uses: cachix/install-nix-action@v30
         with:
-          python-version: '3.12'
-          cache: poetry
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+          install_url: https://releases.nixos.org/nix/nix-2.24.11/install
 
       - name: Get source
         uses: ./.github/actions/get-src
```

actionlint (1.7.7)

description is required in metadata of "GenVM get source" action at "/home/jailuser/git/.github/actions/get-src/action.yaml"

(action)